mountpoint-s3/src/cli.rs

```diff
@@ -227,11 +227,20 @@ pub struct CliArgs {
     #[clap(long, help = "Enable debug logging for AWS Common Runtime", help_heading = LOGGING_OPTIONS_HEADER)]
     pub debug_crt: bool,
 
+    #[cfg(feature = "event_log")]
+    #[clap(
+        long,
+        help = "Write structured event log for failed fuse operations to a directory (allowed to be the same as --log-directory). Note that if `MOUNTPOINT_LOG=off` no events will be emitted.",
```

Comment:
Help text could be simpler. E.g.:
```suggestion
        help = "Write structured event log files to a directory (can be the same as --log-directory)",
```
I'd leave the `MOUNTPOINT_LOG` mention for the docs.

Reply:
hm, I'd say those 2 bits to be deleted from the help message are important (that we only log failed fuse operations and that `MOUNTPOINT_LOG=off` disables the event logging), do you thing it's worth to add an `event log` section in [CONFIGURATION.md](https://github.com/awslabs/mountpoint-s3/blob/main/doc/CONFIGURATION.md#logging) or since it's unreleased better keep it in comments in [mountpoint-s3/src/logging/event_log.rs](https://github.com/awslabs/mountpoint-s3/pull/919/files/cb98dfa433eafbe0c98e3fef2e7c58da4c380c89#diff-07e82ed97e76743ed5dd7709689574da3efb385fdb1b347941b6fc6289f1c8fd)?